scripts/initializeProposal.ts

```diff
@@ -15,37 +16,170 @@ const PANTERA_PUBKEY = new PublicKey(
   "BtNPTBX1XkFCwazDJ6ZkK3hcUsomm1RPcfmtUrP6wd2K"
 );
 
-async function main() {
-  const senderAcc = await token.getOrCreateAssociatedTokenAccount(
+const COST_DEPLOY = 4 * LAMPORTS_PER_SOL
+
+// Transfer
+const transferToAccount = async (
```

Comment:
this is to build a transfer instruction, right? would suggest `buildTreasuryTransferInstruction` or something similar